### ISSUE TITLE
gz-sim: Case study: migrating the ArduPilot ModelPlugin from Gazebo classic to Gazebo

### ISSUE THREAD

**Issue by azeey-test** [gazebosim/gazebo_test_cases#1209]
## Configuration

* os: Ubuntu Noble

## Links

* [Live](https://gazebosim.org/api/sim/9/ardupilot.html)
* [Source](https://github.com/gazebosim/gz-sim/tree/gz-sim9/tutorials/migrating_ardupilot_plugin.md)

**Comment by akky20** [comment]
I'm interested , can I get more info about what to test in this , like do I need to find any typo or any factual error ?

**Comment by iche033** [comment]
yeah, check links, typos, formatting etc. The tutorial points to this repo: https://github.com/gerkey/ardupilot_gazebo/tree/ignition. I just noticed that the code in there is still using ignition and likely won't compile in Ionic. We'll need to add a note in the tutorial to mention this.
